src/clj/fluree/db/json_ld/credential.cljc

```diff
@@ -115,8 +115,22 @@
         auth-did                 (did/auth-id->did id)]
     {:subject (json/parse payload false) :did auth-did}))
 
+(defn is-JWS?
```

Comment:
I think just `jws?` would be a more idiomatic function name.

Reply:
OK. Resolved with 78f1732d6950d41a79de9a561adf640f159b21df